vertx-db2-client/scripts/db2.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+echo "Starting DB2 database with:"
+echo "    user=db2user"
+echo "    pass=db2pass"
+echo "  dbname=vertx_db"
+echo "    port=50000"
+echo ""
+echo "### DB will be usable when you see the following message ###"
```

Comment:
you should add a line of text to say that the DB will run until the user hits ctrl-c to shutdow the docker container